cloud/api-gateway/src/main/resources/application.yaml

```diff
@@ -39,6 +39,14 @@ spring:
               - name: CircuitBreaker
                 args:
                   name: GatewayCircuitBreaker
+          - id: transaction-history-manager
```

Comment:
for now when you do request into `transaction-history` your request doesn't check token (you can check it), you need to add `Authentication` filter, check others for understanding. And check name of application name of your service `transaction-history`, it doesn't have `manager`, just `transaction-history`